<?xml version="1.0"?>
<launch>

    <param name="total_uav" default="2"/>
    <param name="delay_mean" default="0.0001"/>
    <param name="delay_std" default="0.00000001"/>

    <node pkg="sensor_fusion" type="buffer.py" name="delays" required="True" output="screen">
	</node>
    
    <!-- <group ns = "uav0">
        <include file="$(find sensor_fusion)/launch/uav_inbound.launch" >
            <arg name="ID" value="0"/>
        </include>
    </group>

    <group ns = "uav1">
        <include file="$(find sensor_fusion)/launch/uav_inbound.launch">
            <arg name="ID" value="1"/>
        </include>
    </group> -->


</launch>
    
   